src/dev/flang/ast/AbstractFeature.java

```diff
@@ -1020,7 +1020,7 @@ private void existingOrNewTypeFeature(Resolution res, String name, List<Abstract
                     outer().isTypeFeature() ? outer()
                                             : outer().typeFeature(res);
     _typeFeature = res._module.declaredOrInheritedFeatures(outerType,
-                                                           FeatureName.get(name, 0)).getFirstOrNull();
+                                                           FeatureName.get(name, 1)).getFirstOrNull();
```

Comment:
@fridis Yes should probably be `1+typeArgs.size()`

Reply:
@fridis or rather just `typeArgs.size()`
since a few lines below:
```
        var typeFeature = new Feature(p, visibility().typeVisibility(), 0, NoType.INSTANCE, new List<>(name), typeArgs,
                                      inh,
                                      Contract.EMPTY_CONTRACT,
                                      new Impl(p, new Block(new List<>()), Impl.Kind.Routine));
```